src/ReplayTimeline.cc

```diff
@@ -866,12 +866,12 @@ ReplayResult ReplayTimeline::reverse_continue(
           final_result.break_status = BreakStatus();
           return final_result;
         }
-        LOG(debug) << "Seeked backward from " << end << " to " << start;
+        LOG(debug) << "Sought backward from " << end << " to " << start;
```

Comment:
While this is technically correct, using "seeked" as the past tense for a "seek" operation is pretty common. E.g. https://developer.mozilla.org/en-US/docs/Web/Events/seeked. So I'll leave this as-is. Thanks for the rest of the changes!